app/src/test/java/com/futurice/freesound/feature/search/SearchFragmentViewModelTest.java

```diff
@@ -42,46 +43,46 @@
 public class SearchFragmentViewModelTest {
 
     @Mock
-    SearchDataModel searchDataModel;
+    private SearchDataModel searchDataModel;
 
     @Mock
-    Navigator navigator;
+    private Navigator navigator;
 
     private SearchFragmentViewModel viewModel;
 
     @Before
     public void setUp() {
         MockitoAnnotations.initMocks(this);
+
         viewModel = new SearchFragmentViewModel(searchDataModel, navigator);
     }
 
     @Test
     public void getSounds_emitsNone_whenSearchResultsIsNone() {
         new ArrangeBuilder().enqueueSearchResults(Option.none());
 
-        TestObserver<Option<List<DisplayableItem>>> ts = viewModel.getSoundsStream().test();
-
-        ts.assertValue(Option.none());
+        viewModel.getSoundsStream()
```

Comment:
I've started using this format too, where possible. Let's adopt it where possible.